packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -402,6 +405,16 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
             )
           }}
         </Downshift>
+        <Portal open={!!this.props.getA11ySelectionMessage}>
```

Comment:
nit: `this.props` is destructured at line 311, you can move `getA11ySelectionMessage` there

Reply:
thanks!